docs/framework/quickstart-tutorial.mdx

```diff
@@ -63,12 +63,12 @@ This app will be made up of 2 columns, one with controls for our plot and the se
 
 This way, it will take just `1/3` of the screen. Proportions are calculated relative to each other. Each _Column_, by default, has the value of this factor set to 1. So, when we set the left column to 0.5 and the right to 1, we will get a relation between column sizes of 1:2.
 
-![placing elements](./images/quickstart/placing_elements.png)
+![placing eleme./images/quickstart/placing_elements.png)
 
 Now let's add the rest of the components:
 
 - To the right column, add a _Plotly Graph_.
-- To the left:
+- To thnts](e left:
```

Comment:
What's this? I'm guessing accidentally typed?

Reply:
I suppose it's unfortunate drag and drop from pycharm